tests/base_test.py

```diff
@@ -40,6 +41,7 @@ class BaseTest(unittest.TestCase):
 
     auths = {}
     test_uid = "dhf8r"
+    flask_globals = g
```

Comment:
why this obfuscation?

Reply:
Mostly for troubleshooting. I was seeing some weird behavior with the value of g.user being different from one second to the next. So I wanted to make sure they were all referencing the same thing. I figured out the weirdness, so I'll remove the obfuscation before merging.